### DISCUSSION TITLE
下载了新8.0.0版本后无法授权pro，老版的打不开网页

### DISCUSSION THREAD

**Discussion by GithubLexi666** [getlantern/lantern-forum-cn#663] [Q&A]
或许有没有蓝灯用户朋友可以帮帮我，客服也不理人，真的要急死了
![微信图片_20240701221336](https://github.com/getlantern/lantern-forum-cn/assets/167181786/32cc20ca-f0eb-499e-8b04-7932f26bc9d8)

**Comment by MaxMKMax** [comment]
@GithubLexi666 您好，谢谢您的反馈。以后有什么问题来GitHub找我就好. 有时客服那里太多邮件, 回复不来. 在这里反馈是最好的. 蓝灯已经在正视这个问题了。非常抱歉影响了您的体验感。谢谢您支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by XCLlantern** [comment]
解决了吗？我也遇到了这个问题。最近电脑系统崩溃了，重新安装就这样了

**Comment by GithubLexi666** [comment]
你好，不知道你之前安装的是那个版本，我发现就是我必须用我注册时候的那个版本才有用

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年07月02日 15:29 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | GithubLexi666 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 下载了新8.0.0版本后无法授权pro，老版的打不开网页 (Discussion #663) |

解决了吗？我也遇到了这个问题。最近电脑系统崩溃了，重新安装就这样了

**Reply by XCLlantern** [reply]
我已经用了几年了，不知道是哪个版本。我总是在蓝灯更新以后，保存最新的蓝灯应用程序。目前下载最新版安装以后总是闪退，但是安装测试版可以用。新的问题是无法转成专业版，哪怕另一台电脑显示“授权成功”也不行，正在等待他们回复

**Comment by GithubLexi666** [comment]
我也是安装测试版可以用，但是没办法授权成专业版，即使另一台pro device显示授权成功也不行，但我觉得应该还是不兼容的问题。因为我就是换成最初开始的蓝灯之后就不用授权显示该设备已经授权，应该也就是说之前那台pro显示授权成功的只是与我匹配兼容的版本的蓝灯，我建议你也是多下载一些版本试试看，当然这是我的建议啦。

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年07月02日 22:46 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | GithubLexi666 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 下载了新8.0.0版本后无法授权pro，老版的打不开网页 (Discussion #663) |

我已经用了几年了，不知道是哪个版本。我总是在蓝灯更新以后，保存最新的蓝灯应用程序。目前下载最新版安装以后总是闪退，但是安装测试版可以用。新的问题是无法转成专业版，哪怕另一台电脑显示“授权成功”也不行，正在等待他们回复

**Reply by XCLlantern** [reply]
谢谢！你这个建议很有启发性，我明天再试试

**Reply by XCLlantern** [reply]
哇，你这个建议太好了，忍不住马上试了一下。重新安装了windows-7版本的蓝灯，马上就可以通过邮件授权成功。现在已经是正式版了。感谢感谢！

**Reply by XCLlantern** [reply]
哇，你这个建议太好了，忍不住马上试了一下。重新安装了windows-7版本的蓝灯，马上就可以通过邮件授权成功。现在已经是正式版了。感谢感谢！

**Comment by GithubLexi666** [comment]
好耶好耶，很开心能够帮到你，祝你天天开心！

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年07月02日 23:15 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | GithubLexi666 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 下载了新8.0.0版本后无法授权pro，老版的打不开网页 (Discussion #663) |

哇，你这个建议太好了，忍不住马上试了一下。重新安装了windows-7版本的蓝灯，马上就可以通过邮件授权成功。现在已经是正式版了。感谢感谢！

**Reply by XCLlantern** [reply]
感谢感谢！也祝你天天开心！

**Reply by XCLlantern** [reply]
感谢感谢！也祝你天天开心！
